main/factor_lan.R

```diff
@@ -0,0 +1,72 @@
+# Code by Lan Cheng
+library(caret)
+
+# redirect output to a file
+sink("factor.out")
+
+# Factor analysis to pushback metrics to 3-7 factors
+dat <- read.csv("/dev/stdin")
+dat$comb <- 2*dat$mot1 + 3*dat$mot2
```

Comment:
this shouldn't be here, right?

Reply:
Right!!!! That was my test!!! I'll remove it =p